betainc_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"testing"
+
+	"github.com/gonum/floats"
+)
+
+func TestIncBeta(t *testing.T) {
+	tol := 1e-14
+	tol2 := 1e-10
+	// Test against values from scipy
```

Comment:
Is it worth adding a test that tests whether `1 - Incbeta(a,b,x)  =  Incbeta(b,a,1-x)`?